Portal/src/Datahub.Core/Components/AuthViews/DatahubAuthView.razor

```diff
@@ -35,6 +35,9 @@
     [Parameter]
     public string UserGraphId { get; set; }
 
+    [Parameter]
+    public bool ShowToAdmins { get; set; } = true;
```

Comment:
Should the default 'ShowToAdmins' be true? Im going to assume yes, because the intention is to only set it to false on private workspaces where admins need to be given permission to view the workspace contents... but just incase i'm assuming wrong, going to leave this note.

Reply:
I'm open to having it either way.

The parameter is being set to `false` by default on the workspace pages. If there are any uses of `AuthLevels.Workspace...` elsewhere, they'd default to `true` this way.

Before the PR is ready, I'll double check if that happens anywhere and make sure it's addressed.

Reply:
In my opinion it should be the other way... i.e permissions should generally always be suppressed and actively granted when required. However, maybe this is a unique case because it deals admin permissions. We would just have to remember to actively take permissions away from admins should another page with similar requirements be required in the future.

Reply:
I've verified there's no other use. I'll switch it to false by default with my updates later today.